### DISCUSSION TITLE
add 3:G emoji

### DISCUSSION THREAD

**Discussion by drewrsimpson** [runelite/runelite#16517] [Ideas]
Pls add 3:G to Emojis plugin so we can send this to each other for a better gameplay experience 
![image](https://user-images.githubusercontent.com/70524934/233275248-77b9f476-3ae2-4a81-bc8f-dc234d10234a.png)

**Comment by hat-dogs** [comment]
Looks like someone had a similar idea 👀

May I offer my version as well

![Cowboy_Gorilla](https://github.com/runelite/runelite/assets/113656469/d49a22e9-3047-45b7-a464-3775bfe339ad)

**Comment by drewrsimpson** [comment]
Very important pls

**Comment by IdNkI** [comment]
Very important pls adam fix

**Comment by drewrsimpson** [comment]
Bump
